.github/workflows/ci-package.yml

```diff
@@ -59,6 +59,8 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+      with:
+        submodules: true
```

Comment:
The checkout step is used in all the other .yml files too

Reply:
Done.